data/tlg0062/tlg001/tlg0062.tlg001.perseus-eng1.xml

```diff
@@ -0,0 +1,683 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Phalaris</title>
+<author>Lucian of Samosata</author>
+<editor>Austin Morris Harmon, 1878-1950</editor>
```

Comment:
`<editor role="translator">` strike years 1878-1950

Reply:
In the translation files, Harmon has 'role="translator".

He did, however, edit the Greek and so he is just <editor/> in the Greek files.

We can express the different roles because we have split the Greek and the English -- that is an advantage.